src/main/resources/default/templates/system/state.html.pasta

```diff
@@ -3,145 +3,101 @@
-        </w:subHeading>
-
-        <table class="table table-striped">
-            <tr>
-                <td style="overflow: hidden">
-                    Uptime
-                </td>
-                <td class="align-right" style="overflow: hidden">
-                    @info.getUptime()
-                </td>
-            </tr>
+            <t:inlineInfo class="d-none d-md-block">
+                Failed <b>@cluster.countFailedIntervals()</b> of <b>@cluster.getMonitoringIntervals()</b> intervals
+                (Limit:
+                <b>@cluster.getCriticalIntervalLimit()</b>)
+            </t:inlineInfo>
+            <i:block name="actions">
+                <i:if test="all">
+                    <a href="/system/state" class="btn btn-link active mr-2">
+                        <i class="fa fa-plus"></i>
+                        <span class="d-none d-lg-inline">Show all</span>
+                        <a class="btn btn-link mr-2 d-none d-md-inline-block" href="/system/state?all=true"><i
+                                class="fa fa-sync"></i></a>
+                    </a>
+                    <i:else>
+                        <a href="/system/state?all=true" class="btn btn-link mr-2">
+                            <i class="fa fa-plus"></i>
+                            <span class="d-none d-lg-inline">Show all</span>
+                        </a>
+                        <a class="btn btn-link mr-2 d-none d-md-inline-block" href="/system/state"><i
+                                class="fa fa-sync"></i></a>
+                    </i:else>
+                </i:if>
+            </i:block>
+        </t:pageHeader>
+    </i:block>
 
-            <i:for type="sirius.kernel.health.metrics.Metric" var="m" items="info.getMetrics()">
+    <t:datacards>
+        <t:datacard title="@CallContext.getNodeName()" subTitle="@apply('Uptime %s',  uptime)">
+            <i:for type="sirius.kernel.health.metrics.Metric" var="m" items="metrics.getMetrics()">
                 <i:if test="all || m.getState() != sirius.kernel.health.metrics.MetricState.GRAY">
-                    <tr class="@m.getState()">
-                        <td style="overflow: hidden">
-                            @m.getLabel()
-                        </td>
-                        <td class="align-right" style="overflow: hidden">
-                            @m.getValueAsString()
-                        </td>
-                    </tr>
+                    <t:infoLine label="@m.getLabel()">
+                        <span class="text-sirius-@(m.getState().name().toLowerCase())-dark">@m.getValueAsString()</span>
+                    </t:infoLine>
                 </i:if>
             </i:for>
-        </table>
 
-    </i:for>
-</w:page>
+            <i:block name="footer">
+                <i:switch test="@cluster.getNodeState().name()">
+                    <i:block name="GRAY">
+                        <t:dot color="green">Norminal</t:dot>
```

Comment:
```suggestion
                        <t:dot color="green">Normal</t:dot>
```

Reply:
Aber spaceX und NASA dürfen dass au immer so schreiben, wenn se Raketen starten :(

Reply:
In dem Fall drücke ich ein Auge zu